src/component/missionProfile.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import { useSelector } from "react-redux";
```

Comment:
- [ ] For best practices and as required for all Microverse projects, kindly make sure you fix your linters errors and not disable them.